docs/stake-transition.md

```diff
@@ -0,0 +1,147 @@
+# Transition of Existing Stakes to the Babylon Chain
+
+This document is intended for Bitcoin stakers who have
+submitted confirmed Bitcoin staking transactions
+on the Bitcoin blockchain before the launch of
+the phase-2 Babylon blockchain (e.g., via participation
+in one of the phase-1 networks).
+It outlines the eligibility criteria for the registration of
```

Comment:
```suggestion
It outlines the eligibility criteria for the transition of
```

Should we stick with either `transition` or `registration`? They are both used and I feel a bit confused. Or we can add one line to say they are interchangeable

Reply:
I prefer using registration instead of transition, as transition might imply that the funds are transferred from Bitcoin to somewhere else. Will update transition elsewhere to registration